.ippen.media.txt

```diff
@@ -0,0 +1,58 @@
+# Author: HolgerAusB
+#
+# ippen.media is a German newspaper network that operates about 60+ websites
+# with (local) news and magazines, all or most of the newspaper sites with the same CMS.
+# magazine sites are not working yet.
+#
+#==========================
+#
+# If you host fultext-rss/wallabag on your own servers you may
+# a)
+# copy this file and name it for your domains e.g.:
+#        custom/hanauer.de.txt
+#        custom/ruhr24.de.txt
```

Comment:
Looks like there can be a fingerprints from the html code.
```
/\<div\s*class=([\'"])id-SiteBEEPWrap([\'"])\>/i
```

That can be added over here: https://github.com/j0k3r/graby/blob/6ed01641205b7fbdea4ce9ba15a34989af9edc1a/src/Extractor/ContentExtractorConfig.php#L39

What do you think?

Reply:
will check that tomorrow. Thank you for looking into that

Reply:
this one is not working with fulltext-rss. As I wrote in #926 ftr seems to check only the `<head>` for matches, but your suggestion is within the `<body>`. And to get sure, I also used a shorter version just for testing:
`'id-SiteBEEPWrap' => array('hostname'=>'fingerprint.ippen.media', 'head'=>true),`
But still got "No fingerprint matches"

Reply:
@HolgerAusB We'll see if it's possible to add this to Full-Text RSS in the config file. It should be possible to use the body and not just the head, if not, we'll fix that in the next version.

Reply:
Sorry for the slow response. Haven't forgotten about this, just been very busy. I think we can merge this, as the config changes if/where they're needed have to happen separately anyway. What do you think @j0k3r @HolgerAusB